### DISCUSSION TITLE
Create a set-up script for individual setups

### DISCUSSION THREAD

**Discussion by torwag** [JensenNick/huawei_solar_pees#53] [Forum]
Trying the PEES system, I adapted it to my setup of three inverters, each connected by a battery. 
I tried to get the logic and figure out where to add inverter 3 and which sensors to add to which calculations.
That is a rather repetitive but also error-prone task.
I'm wondering, if we could create a simple script to ask people how many inverters they own, how many batteries, the currency, and all kinds of questions which might be different in particular set-ups thus, the script will generate the PEES config file accordingly. 
That would also help people with updating, as they might simply rerun the new script to create the new config.

**Comment by Roving-Ronin** [comment]
Hi @torwag 

Why thank you for volunteering for this task.  ;-) LoL

Scripting is something that I've already raised as being nice to have, my focus was in relation to detecting the inverters / batteries etc and auto configuring the PEES Stats package, instead of the current manual inputs... however it depends upon working out if / how to do so currently (and as we have free time available).

See (unofficial / playing with ideas roadmap) from chatting with Nick:  https://github.com/JensenNick/huawei_solar_pees/discussions/51

**Comment by JensenNick** [comment]
@torwag I understand your situation and agreed that would be great, but I am afraid scripting is beyond my abilities.
